### ISSUE TITLE
Environments for postman runner

### ISSUE THREAD

**Issue by mariusstaicu** [kubeshop/testkube#806]
I would like to be able to switch to running postman tests with testkube in k8s, however this seems not to be possible because testkube postman runner does not support postmant environments and environment variables (or is it ?).

Is there any plan to support this in the future ?

**Comment by exu** [comment]
Hi @mariusstaicu for now we're supporting only passing env parameters which are passed to postman executor e.g.
```
kubectl testkube scripts start SOME_SCRIPT -p param1=val1 -p param2=val2
```
(but there is no possibility to pass whole env file) 

Can you elaborate a little what would be your use case.  Do you want to pass whole file with env values ?

**Comment by mariusstaicu** [comment]
Hi @exu,
I have multiple collections already using env variables, so I wanted to see if it is feasible to run them using testkube.
With postman runner you can specify an entire env.json file as parameter which has more env variable inside:
`newman run -e env.json ... `

I didn't know about -p parameter for script start, so I guess I could use that instead.

Thanks

**Comment by exu** [comment]
@mariusstaicu yeah - we are creating this file for the runner in the executor to pass those `-p` parameters.

so to clarify: 
you would like to pass an additional file with envs to `kubectl testkube scripts start testname --params-file='my-envs.json' 
I think we could think about merging them together e.g. env-file overriden by params passed in flags in case of same keys.

does it have sense for you ?

**Comment by olensmar** [comment]
would it make sense to also be able to pass parameters to an executor from a ConfigMap and/or Secret!?

**Comment by pnx472** [comment]
There is a `--params-file` flag in cli but it doesn't seem to work. It would be nice to have an option to pass postman environment json file.

**Comment by exu** [comment]
Hi @pnx472 we'll look at it and back to you soon

**Comment by pnx472** [comment]
Regarding `--params-file`, maybe this is the issue, use of param [executionParamsFileContent](https://github.com/kubeshop/testkube/blob/b1d07765f65959a173b6f6eb362eed9f851caf57/pkg/api/v1/client/proxy.go#L185) has not been implemented yet.

**Comment by exu** [comment]
Yes indeed :) good catch - I've added PR which fixes this one #995
